hw/ip/kmac/dv/kmac_base_sim_cfg.hjson

```diff
@@ -99,6 +99,7 @@
     {
       name: "{name}_sideload"
       uvm_test_seq: kmac_sideload_vseq
+      run_opts: ["+test_timeout_ns=1_000_000_000"]
```

Comment:
Yes, this is necessary because kmac now drains all data beats on the HW app interface when there's an error.

Re this being very long -- time is relative, right? :-) kmac block level DV simulates at a relatively high rate (simulated time over wall clock time), and other tests have an even higher timeout (e.g., `kmac_long_msg_and_output` above has 10 s).